### PR TITLE
[TSK-254] add UserManager to test user-manager

### PR DESCRIPTION
**Pull request by Hoocki**

### CODE REVIEW

**Comment by Hoocki** [comment]
image

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import static io.gatling.javaapi.http.HttpDsl.http;
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+public class UserManager extends Simulation {
+
+    HttpProtocolBuilder httpProtocol = http
+            .baseUrl("http://localhost:8080/")
+            .acceptHeader("application/json");
+
+    ScenarioBuilder scn = scenario("Getting users ")
```
Comment (adsukharev) [inline gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java]:
```suggestion
    ScenarioBuilder scn = scenario("Getting users")
```

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    {
+        setUp(
+                scn.injectOpen(
+                        rampUsersPerSec(0).to(200).during(Duration.of(10, SECONDS)),
+                        rampUsers(230).during(Duration.of(30, SECONDS))
+                )
+        ).protocols(httpProtocol)
+                .assertions(
+                        global().successfulRequests().percent().gte(99d)
```
Comment (adsukharev) [inline gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java]: global().responseTime().max().lte(1000), // 1 sec
global().successfulRequests().percent().gte(99.999));

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/GatlingRunner.java [hunk 3]
```diff
@@ -10,7 +10,7 @@ public static void main(final String[] args) {
                 .resourcesDirectory("src/gatling/resources")
                 .resultsDirectory("build/reports/gatling")
                 .binariesDirectory("build/classes/java/gatling")
-                .simulationClass("com.qpay.gatlingloadtest.simulation.UserSimulation");
+                .simulationClass("com.qpay.gatlingloadtest.test.simulation.UserManager");
```
Comment (adsukharev) [inline gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/GatlingRunner.java:13]:
```suggestion
.simulationClass("com.qpay.gatlingloadtest.simulation.UserManager");
```

**Review by adsukharev — APPROVED** [review]

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/GatlingRunner.java (hunk 3 above)
Comment (adsukharev) [reply gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/GatlingRunner.java:13]: @Hoocki You did not do this change. Test is broken

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java (hunk 2 above)
Comment (adsukharev) [reply gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/test/simulation/UserManager.java]: @Hoocki You did not add this line `global().responseTime().max().lte(1000)`

File: gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/simulation/UserManager.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    ScenarioBuilder scn = scenario("Getting users")
+            .exec(http("get customers request")
+                    .get("/api/v1/customers/26")
+                    .headers(Map.of("Authorization", "Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1c2VyMzNAZ21haWwuY29tIiwiaWF0IjoxNzExMzU5NzE2fQ.toUjnetk8WCZB_CyKZKrMs4BITFzJBewzV6EFLpN3iE"))
+            );
+
+    {
+        setUp(
+                scn.injectOpen(
+                        rampUsersPerSec(0).to(200).during(Duration.of(10, SECONDS)),
```
Comment (adsukharev) [inline gatling-load-test/src/gatling/java/com/qpay/gatlingloadtest/simulation/UserManager.java:32]: Start with at least 10 requests.

**Pull request merged**